target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8065-r7800.dts

```diff
@@ -343,7 +363,10 @@
 			compatible = "virtual,mdio-gpio";
 			#address-cells = <1>;
 			#size-cells = <0>;
-			gpios = <&qcom_pinmux 1 0 &qcom_pinmux 0 0>;
+			gpios = <&qcom_pinmux 1 GPIO_ACTIVE_HIGH &qcom_pinmux 0 GPIO_ACTIVE_HIGH>;
+			pinctrl-0 = <&mdio0_pins &rgmii2_pins>;
```

Comment:
OK change it to place like in c2600 :)

I think it does not matter where it is, because it only sets the state on gpio pins,
gpios line set pins to use.